README.md

```diff
@@ -215,13 +286,17 @@ Make the necessary changes to your dockerfile with respect to your new changes.
 | database.indices                | index or collection name to be referred to from vector database based on input audienceType    |                                      |
 | database.top_docs_to_fetch      | Number of filtered documents retrieved from vector database to be passed to Gen AI as contexts | 5                                    |
 | database.docs_min_score         | Minimum score of the documents based on which filtration happens on retrieved documents        | 0.4                                  |
+| redis.ttl         | Redis cache expiration time for a key in seconds.        | 43200                               |
```

Comment:
Is this used for the conversation history? can we capture the info here

Reply:
Done